src/main/java/com/imsweb/datagenerator/naaccr/NaaccrDataGenerator.java

```diff
@@ -29,29 +29,7 @@
 import com.imsweb.datagenerator.naaccr.rule.patient.SexRule;
 import com.imsweb.datagenerator.naaccr.rule.patient.SsnRule;
 import com.imsweb.datagenerator.naaccr.rule.patient.VitalStatusRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.AddressAtDxRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.AgeRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.CensusRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.CollaborativeStageRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.DateOfConclusiveDxRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.DateOfDiagnosisRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.DateOfInitialRxRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.DateOfLastContactRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.DxConfirmationRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.MaritalStatusRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.MultiTumorsRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.NapiiaRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.NhiaRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.PrimaryPayerRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.ReportingSourceRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.RxSummaryRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.SeerCodingSystemRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.SeerRecordNumberRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.SeerTypeOfFollowUpRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.SequenceNumberCentralRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.SiteRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.TumorMarkerRule;
-import com.imsweb.datagenerator.naaccr.rule.tumor.TumorRecordNumberRule;
+import com.imsweb.datagenerator.naaccr.rule.tumor.*;
```

Comment:
We don't use * imports, that's because you are not using the SEER coding style (as you pointed out yourself). I don't like these changes, so please import the coding styles. I am attaching them to this comment.
[seer-intellij-settings-20170209.zip](https://github.com/imsweb/data-generator/files/842610/seer-intellij-settings-20170209.zip)